pkg/services/featuremgmt/registry.go

```diff
@@ -1297,5 +1297,13 @@ var (
 			RequiresRestart: true,
 			Created:         time.Date(2024, time.January, 3, 12, 0, 0, 0, time.UTC),
 		},
+		{
+			Name:         "enablePluginsTracingByDefault",
+			Description:  "Enable plugin tracing for all external plugins",
+			FrontendOnly: false,
+			Stage:        FeatureStageExperimental,
+			Owner:        grafanaPluginsPlatformSquad,
+			Created:      time.Date(2024, time.January, 9, 12, 0, 0, 0, time.UTC),
```

Comment:
Not sure it's really necessary, but should we be setting `RequiresRestart` to `true` here?

Reply:
Good spot, I have added it :+1: